source/mobile/deploy-mobile-apps-using-emm-provider.rst

```diff
@@ -0,0 +1,131 @@
+
+Enroll Devices
+--------------
+
+When building your own custom versions of Mattermost Mobile apps, consider your organization’s mobile policy:
+
+- Can users bring their own device (BYOD) If so, what devices will be used?
+- Are devices company-owned and company-issued?
+- Are both options supported?
+- What operating systems do you want to start testing?
+
+Once you know what possible device configurations you’ll be supporting, consider creating a sample configuration, then running validation tests against each configuration item.
+
+Generate and Assign Device Profiles
+-----------------------------------
+
+Generate and assign a device profile for device-wide configurations through the EMM provider.
+
+Consider Mobile VPN Options
+---------------------------
+
+A Virtual Private Network (VPN) allows a device outside a firewall to access content inside the firewall as if it were on the same network.
+
+.. note::
+
+    Some mobile VPN options depend on the requirements of your organization and the demands and/or the needs of your users. 
+
+We recommend one of two options: `per-app VPN <#per-app-vpn>`_ or a `device VPN <#device-vpn>`_ to secure your deployment. Both options are compatible with most EMM providers. 
+
+We also recommend following our `recommended steps to secure your deployment <https://docs.mattermost.com/mobile/mobile-appstore-install.html>`__ and to review the following commonly-asked questions about data security on mobile devices:
+
+- `How data is handled on a device after an account is deleted? <https://docs.mattermost.com/mobile/mobile-faq.html#how-is-data-handled-on-mobile-devices-after-a-user-account-is-deactivated>`__
+- `What post metadata is sent in mobile push notifications? <https://docs.mattermost.com/mobile/mobile-faq.html#what-post-metadata-is-sent-in-mobile-push-notifications>`__
+- `What are my options for securing the Mobile apps? <https://docs.mattermost.com/mobile/mobile-faq.html#what-are-my-options-for-securing-the-mobile-apps>`__
+- `What are my options for securing push notifications? <https://docs.mattermost.com/mobile/mobile-faq.html#what-are-my-options-for-securing-push-notifications>`__
+
+Per-app VPN
+~~~~~~~~~~~
+
+A common approach is to use a per-app VPN. This provides a connection to the VPN when needed (on-demand). If using a per-app VPN with Mattermost, you can configure the following options:
+
+- **useVPN**: Mattermost waits until the connection to the VPN server is established before making any requests (otherwise they will fail). Only supported on iOS given Android OS cannot support waiting - still works but the first connection attempt may fail. 
```

Comment:
```suggestion
- **useVPN:** Mattermost waits until the connection to the VPN server is established before making any requests (otherwise they will fail). This is only supported on iOS as Android OS cannot support waiting. It still works but the first connection attempt may fail.
```